feat: UTC-371: Low Agreement Resolution section in Quality Settings

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/19172443821)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/664
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19142831102

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19171895223)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19171895223

mcanu committed:
Merge branch 'develop' into 'fb-utc-371'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19171895223

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19172443821

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/19173903608
